### ISSUE TITLE
[Bug]: When starting DBBrowser a messagebox (Could not get the InetworkConnection instance for the adapter GUID) is displayed 

### ISSUE THREAD

**Issue by StephanSchwebel** [sqlitebrowser/sqlitebrowser#3783]
### What did you do?

I started "DB Browser for SQLite.exe"

### What did you expect to see?

The user interface being launched without interruption

Additional information:
Until today, I have used a previous version of DB Browser (not sure which) and using this version the user interface launched without this message box.

### What did you see instead?

A message box saying: "Could not get the InetworkConnection instance for the adapter GUID"
![Screenshot 2024-10-21 101959](https://github.com/user-attachments/assets/42734389-ceb7-484b-902a-84d5d6b0fc84)

### DB4S Version

3.13.1

### What OS are you seeing the problem on?

Windows

### OS version

Windows 11

### Prevention against duplicate issues

- [X] I have searched for similar issues

**Comment by justinclift** [comment]
Yeah, there have been a few instances of that showing up.  If you turn off the automatic check for new versions (in the application preferences) then it'll stop showing up when you start the application.

That's probably the best workaround for now, until we have time/effort available to investigate it. :smile:
